README.md

```diff
@@ -98,19 +98,24 @@ You can install the extension [in the marketplace here](https://marketplace.visu
 A. Either:
 - You have no workspace open and opened a preview.
 - You opened a preview for a file that is not a part of your workspace(s).
+- You have the `livePreview.serverRoot` is set to a sub-folder in your workspace and your file is not a child of that subfolder.
```

Comment:
Should be "You have `livePreview.serverRoot` set to" or "You have the `livePreview.serverRoot` setting set to"